Mage.Client/src/main/java/org/mage/plugins/card/dl/sources/GathererSymbols.java

```diff
@@ -119,8 +119,8 @@ protected DownloadJob computeNext() {
                         case "S":
                             symbol = "snow";
                             break;
-                        case "P":
-                            symbol = "p";
+                        case "H":
+                            symbol = "h";
```

Comment:
is this actually a valid symbol in gatherer now?

Reply:
A couple people [here](https://github.com/magefree/mage/issues/11853#issuecomment-2254226691) said we should be good to do this before wizards gets this symbol updated. As of now, {P} leads to the pawprint symbol, and {H} is not a valid link.

Reply:
Yeah, this change doesn't depend on gatherer because scryfall has it, but if the gatherer part isn't working, then best to add a comment here